matchbox_demo/src/main.rs

```diff
@@ -49,12 +52,6 @@ fn main() {
         // make it happen in the bevy app
         .build(&mut app);
 
-    // (optional) Enable debug log level for matchbox
-    app.insert_resource(LogSettings {
-        filter: "info,wgpu_core=warn,wgpu_hal=warn,matchbox_socket=debug".into(),
-        level: bevy::log::Level::DEBUG,
-    });
```

Comment:
**note** `LogSettings` was replaced with settings in `LogPlugin`. When I added that plugin myself I got an error that it was already configured. I didn't dig further into this.

Reply:
You can call `.set` on `DefaultPlugins` instead.